releases/NS30_GP/6.11/ns30gp.metainfo.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Richard Hughes <richard@hughsie.com> -->
+<!-- Copyright 2018 Mario Limonciello <mario.limonciello@dell.com> -->
+<component type="firmware">
+  <id>com.8bitdo.ns30-gp.firmware</id>
+  <name>N30 Pro 2</name>
+  <summary>Firmware for the 8Bitdo NS30 GP Game Controller</summary>
+  <description>
+    <p>
+    </p>
+  </description>
+  <provides>
+    <!-- USB\VID_2dC8&PID_5750 -->
+    <firmware type="flashed">b02f1319-c1c4-54f4-b7f2-5bb3552f6a9e</firmware>
```

Comment:
isnt there a bootloader guid here too?

Reply:
That's exactly bootloader. I'm not sure about another, let me check

Reply:
Are you talking about `78acf201-a624-5c0b-a521-e204be14e3d5`? How should I define it? TBH, I've just cloned the M30 metadata more or less https://github.com/hughsie/8bitdo-firmware/blob/master/releases/M30/1.13/m30.metainfo.xml

Reply:
Richard, any update?

Reply:
Don't you need to add a runtime GUID here as well? Otherwise fwupd isn't going to know the device is a NS30 just from the bootloader.

Reply:
According to https://github.com/fwupd/8bitdo-firmware/issues/25#issuecomment-584759000 the bootloader GUID should **not** be listed in this file, only the runtime GUID.

Reply:
Hello dear gentlemen. Sorry for the long silence, but the device has gone from my hands for a while. Unfortunately, `sudo fwupdmgr get-devices --show-all-devices` doesn't show the gamepad unless it's in bootload mode. I don't know why, but this is the state..